### PR TITLE
test: fix flaky RemoveSortableColumnsIT

### PR DESCRIPTION
**Pull request by sissbruecker**
## Description

Work in progress

Fixing flaky RemoveSortableColumnsIT introduced by #468 

## Type of change

- [x] Internal

### CODE REVIEW

**Review by manolo — APPROVED** [review]

File: vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/grid/it/RemoveSortableColumnPage.java [hunk 1]
```diff
@@ -140,7 +140,6 @@ public RemoveSortableColumnPage() {
         grid.setHeaderGenerator(getHeaderGenerator());
 
         add(grid);
-        grid.redraw();
 
         Button sort = new Button("Sort");
         sort.setId("sort");
```
Comment (tulioag) [inline vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/grid/it/RemoveSortableColumnPage.java:145]: I suggest you also change this id to something different. 
This is a test I did in a random test view.

Screenshot from 2021-05-05 19-58-03

**Review by tulioag — CHANGES_REQUESTED** [review]

File: vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/grid/it/RemoveSortableColumnPage.java (hunk 1 above)
Comment (sissbruecker) [reply vaadin-grid-flow-parent/vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/grid/it/RemoveSortableColumnPage.java]: Thanks Tulio that actually was the issue 👍. Strangely that only showed up when running the combined tests. Anyway it's fixed now.

**Review by tulioag — APPROVED** [review]

**Comment by manolo** [comment]
Hi @sissbruecker , this commit cannot be picked to 19.0 by this bot, can you take a look and pick it manually?

**Comment by sissbruecker** [comment]
I think this will be included as part of Tulios effort to backport https://github.com/vaadin/flow-components/pull/468 to 19. Is that orrect @tulioag ?

**Comment by tulioag** [comment]
It's already merged https://github.com/vaadin/flow-components/pull/931

**Pull request merged**